demos/text_detection_demo/cpp/text_detection_demo.hpp

```diff
@@ -55,6 +56,7 @@ static const char decoder_bandwidth_message[] = "Optional. Bandwidth for CTC bea
 DEFINE_bool(h, false, help_message);
 DEFINE_string(m_td, "", text_detection_model_message);
 DEFINE_string(m_tr, "", text_recognition_model_message);
+DEFINE_string(dt, "", text_recognition_decoder_type_message);
```

Comment:
```suggestion
DEFINE_string(dt, "ctc", text_recognition_decoder_type_message);
```

If provide default option for decoder, old text recognition models (12 & 13) could be used as is (without setting additional parameters). 
Also, if not provide default choice, documentation for text-recognition-0012 should also be changed.